.travis.yml

```diff
@@ -5,57 +5,46 @@ os:
 
 cache:
   directories:
-    - $HOME/.composer/cache
+    - "$HOME/.composer/cache"
 
 env:
- global:
-  - URL=http://192.168.2.99
-  - DB_TYPE=pdo.mysql
-  - DB_HOST=localhost
-  - DB_USER=root
-  - DB_PASS=
-  - DB_PCONNECT=0
-  - DB_NAME=icms
-  - DB_CHARSET=utf8
-  - DB_COLLATION=utf8_general_ci
-  - DB_PREFIX=icms
-  - DB_SALT=icms
+  global:
+    - URL=http://192.168.2.99
```

Comment:
Is this IP used for a specific reason?

Reply:
I can't remember correctly but right now I think that is just random IP to make sure URL environment variable is not empty. That there will be no warnings in tests about this.